stock_inventory_verification_request/views/stock_slot_verification_request_view.xml

```diff
@@ -27,6 +27,9 @@
         <field name="arch" type="xml">
             <form string="Stock Slot Verification Request">
                 <header>
+                    <button name="action_create_inventory_adjustment"
+                            type="object" states="open" string="Create Inventory Adjustment"
+                            class="oe_highlight" />
```

Comment:
maybe do not display the button for done and cacel states?

Reply:
With the states attribute it will only be displayed when In Progress

Reply:
I'm blind, didn't see it :laughing: